dockerfiles/theia/README.md

```diff
@@ -1,33 +1,101 @@
 # How to Build Theia Image
 
 ## Build image manually using build scripts
-To build Theia image manually You could use build.sh scripts.
-Dockerfile uses multi-stage build feature, so first of all You need build builder image -
-theia-dev:
 
-Example:
+To build all the images you can easily run [build.sh](../../build.sh) script located in the repository root.
 
-```shell
-$ ../theia-dev/build.sh
+CI for checking any pull request in this repository runs this script with `--pr` directive
+```bash
+`build.sh --pr`.
+```
+
+**Note**: `--pr` will build only [limited set](../../docker_image_build.include) of docker images.	
+
+To publish docker images use `--push` directive
+```bash
+`build.sh --push`
+```	
+
+## How to build own Che Theia image with Docker
+
+First you need to build `che-theia-dev` image:	
```

Comment:
I don't think it is mandatory. Could you explain when it is needed to be execute?

Reply:
To me we have to put at the top the commands that people would execute 90% of the time.
Building che-theia-dev should go down in the file in a more advance or FAQ section.

Reply:
Section has been reworked